contrib/deploy_timesketch.sh

```diff
@@ -114,9 +114,15 @@ echo "* Installation done."
 
 echo
 echo "Start the system:"
+
 echo "1. cd timesketch"
+cd timesketch
 echo "2. docker-compose up -d"
+docker-compose up -d
```

Comment:
I'm not sure everyone wants to automatically start the containers. Could we put this behind a flag instead?

`--start-containers`

Reply:
Where do input this flag? Can we ask user whether to start container after installation is done?

Reply:
I think both would be fine:
```sh deploy_timesketch.sh —start-container```

Or if that is not set at the end of the script ask
```Would you like start your container (y/n) [default:no]```

What do you think @avitomar12 ?

Reply:
Yes. I also think its better to provide both options to the user.